great-idea-website/css/index.css

```diff
@@ -63,4 +63,169 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+nav {
+	width: 600px;
+	height:30px;
+	box-sizing:border-box;
+	margin-left:100px;
+	word-spacing: 40px;
+	color:grey;
+	margin-bottom:40px;
+	font-size:15px;
+	margin-top:50px;
+}
+	
+.logo {
+	width: 200px;
+	height:30px;
+	box-sizing:border-box;
+	position:absolute;
+	Margin-right:auto;
+	margin-left:10px;
+	margin-top:45px;
+	
+	
+}
+
+ nav, .logo{
+	display:inline-block;
+	vertical-align:text-top; 
+	 
+}
+
+.main-header {
+	text-align: center;
+	margin-left:170px;
+	box-sizing:border-box;
+	margin-top: 40px;
+    font-size:70px;
+	margin-right: 60px;
+	
+}
+
+
+.main-header, .Image-of-a-code-snippet{
+	display:inline-block;
+	vertical-align:top;
+	
+}
+hr {
+	width:810px;
+	margin-left:100px;
+}
+.features {
+	width:400px;
+	height:20px;
+	margin-left:100px;
+	
+}
+.about {
+	width:400px;
+	height:20px;
+	margin-left: 10px;
+	
+}
+
+.about, .features {
+	display: inline-block;
+	
+}
+
+.p-features {
+	width:400px;
+	height:110px;
+	margin-left:100px;
+	box-sizing:border-box;
+	
+}
+.p-about {
+	width:400px;
+	height:110px;
+	margin-left:10px;
+	box-sizing:border-box;
+}
+.p-features, .p-about {
+	display: inline-block
+}
+
+.middle-img {
+	Width:810px;
+	Height:182px;
+	margin-left:100px;
+	margin-bottom:20px;
+}
+
+.services {
+	width: 260px;
+	height: 20px;
+	margin-left:100px;
+}
+
+.product {
+	width: 260px;
+	height: 20px;
+	margin-left:10px;
+}
+.vision {
+	width: 260px;
+	height: 20px;
+	margin-left:10px;
+}
+
+
+.services, .product, .vision {
+	display:inline-block;
+}
+
+.p-services {
+	width: 260px;
+	height: 130px;
+	margin-left: 100px;
+}
+
+.p-product {
+	width: 260px;
+	height: 130px;
+	margin-left: 10px;
+}
```

Comment:
If you pair up your `h3`'s and `p`'s into their own divs, you can just change the margin of them as a group, instead of individually changing their margins for each one.